__tests__/app.test.js

```diff
@@ -11,4 +12,108 @@ describe('backend-top-secrets routes', () => {
   afterAll(() => {
     pool.end();
   });
+
+  it('signs up a user w/ post', async () => {
+    const res = await request(app)
+      .post('/api/v1/user/signup')
+      .send({ email: 'miklo@test.com', password: 'ilovetreats' });
+
+    expect(res.body).toEqual({
+      id: expect.any(String),
+      email: 'miklo@test.com',
+    });
+  });
+  it('signs in an existing user', async () => {
+    const user = await UserService.create({
+      email: 'miklo@test.com',
+      password: 'ilovetreats',
+    });
+
+    const res = await request(app)
+      .post('/api/v1/user/session')
+      .send({ email: 'miklo@test.com', password: 'ilovetreats' });
+
+    expect(res.body).toEqual({
+      message: 'Successfully signed in',
+      user,
+    });
+  });
+
+  it('protects routes using authenticate', async () => {
+    const agent = request.agent(app);
+    await UserService.create({
+      email: 'miklo@test.com',
+      password: 'ilovetreats',
+    });
+
+    let res = await agent.get('/api/v1/user/private');
+    expect(res.status).toEqual(401);
+
+    await agent
```

Comment:
it looks like you log in here but then test a route that only serves to tell everyone who hits it that they aren't logged in.